app/services/organizations/create_service.rb

```diff
@@ -0,0 +1,71 @@
+# class to create a new organization, user, staff account with role admin, and send email via the console
+# call with Organizations::CreateService.new.signal(args)
+# args:
+# {
+#   organization_name: 'value',
+#   organization_slug: 'value',
+#   user_email: 'value',
+#   user_first_name: 'value',
+#   user_last_name: 'value'
+# }
+
+class Organizations::CreateService
+  def signal(args)
+    ActiveRecord::Base.transaction do
+      create_organization(args[:organization_name], args[:organization_slug])
+      create_user(args[:user_email], args[:user_first_name], args[:user_last_name])
+      create_staff_account
+      add_admin_role_to_staff_account
+      send_email
+    end
+  rescue => e
+    puts "An error occurred: #{e.message}"
```

Comment:
```suggestion
    raise "An error occurred: #{e.message}"
```
So it is really loud when the error occurs.